server/routerlicious/packages/lambdas/src/scriptorium/lambda.ts

```diff
@@ -114,15 +144,18 @@ export class ScriptoriumLambda implements IPartitionLambda {
 			() => {
 				this.current.clear();
 				status = ScriptoriumStatus.ProcessingComplete;
+				metric?.setProperty("timestampProcessingComplete", Date.now());
 
 				// checkpoint batch offset
 				try {
 					// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 					this.context.checkpoint(batchOffset!);
 					status = ScriptoriumStatus.CheckpointComplete;
+					metric?.setProperty("timestampCheckpointComplete", Date.now());
```

Comment:
Do we need a similar property for tracking when checkpoint starts?

Reply:
I dont think thats needed because it will be the same as "timestampProcessingComplete". The duration between timestampProcessingComplete and timestampCheckpointComplete would be the time taken to checkpoint.